### PR TITLE
Ecommerce Mutations Updates

### PR DESCRIPTION
**Pull request by wesbos**
Just a few updates to the ecomm example. 

I'm hitting one issue with the server side creation of an array of orderItems

### CODE REVIEW

File: examples-next/ecommerce/mutations.ts [hunk 1]
```diff
@@ -42,34 +49,40 @@ export const extendGraphqlSchema = graphQLSchemaExtension({
...
         });
-
+        console.dir(orderItems, { depth: null });
         // 5. create the Order
-        console.log('Creating the order');
         const order = await context.lists.Order.createOne({
           data: {
             total: charge.amount,
             charge: `${charge.id}`,
+            // TODO create array of items causes error
```
Comment (wesbos) [inline examples-next/ecommerce/mutations.ts:84]: @timleslie There create: orderItem (array of orderItem type) doesn't seem to work. 

Should I first create the orderItems, save their ids, and then connect them instead?

File: examples-next/ecommerce/mutations.ts [hunk 2]
```diff
@@ -42,34 +49,40 @@ export const extendGraphqlSchema = graphQLSchemaExtension({
...
-        const orderItems = User.cart.map((cartItem: any) => {
+        // TODO Type CartItem and OrderItem. Can they be generated?
+        const orderItems = User.cart.map((cartItem) => {
           const orderItem = {
             name: cartItem.product.name,
             description: cartItem.product.description,
             price: cartItem.product.price,
             quantity: cartItem.quantity,
-            image: { connect: { id: cartItem.product.image.id } },
+            image: { connect: { id: cartItem.product.photo.image.id } },
```
Comment (timleslie) [inline examples-next/ecommerce/mutations.ts:73]: Should this be:

```suggestion
            image: { connect: { id: cartItem.product.photo.id } },
```

File: examples-next/ecommerce/mutations.ts [hunk 3]
```diff
@@ -42,34 +49,40 @@ export const extendGraphqlSchema = graphQLSchemaExtension({
...
+        console.dir(orderItems, { depth: null });
         // 5. create the Order
-        console.log('Creating the order');
         const order = await context.lists.Order.createOne({
           data: {
             total: charge.amount,
             charge: `${charge.id}`,
+            // TODO create array of items causes error
+            // message":"Unable to create and/or connect 2 Order.items<OrderItem>
             items: { create: orderItems },
```
Comment (timleslie) [inline examples-next/ecommerce/mutations.ts:86]: Try:

```suggestion
            items: { create: orderItems.map(x => ({ data: x })) },
```

File: examples-next/ecommerce/seed-data/index.ts [hunk 4]
```diff
@@ -1,16 +1,17 @@
 import { products } from './data';
 
 export async function insertSeedData(keystone: any) {
-  console.log('------------ INSERTING DUMMY DATA ------------');
+  console.log(`🌱 Inserting Seed Data: ${products.length} Products`);
   const { mongoose } = keystone.adapters.MongooseAdapter;
   for (const product of products) {
+    console.log(`  🛍️ Adding Product: ${product.name}`);
```
Comment (JedWatson) [inline examples-next/ecommerce/seed-data/index.ts:7]: ❤️ nice use of emojis here!

**Comment by timleslie** [comment]
All the relevant changes from this branch have now been incorporated through other PRs, so closing this out now 👍

**Pull request closed without merging**